pilot/pkg/bootstrap/mesh.go

```diff
@@ -99,6 +99,13 @@ func (s *Server) initMeshConfiguration(args *PilotArgs, fileWatcher filewatcher.
 // initMeshNetworks loads the mesh networks configuration from the file provided
 // in the args and add a watcher for changes in this file.
 func (s *Server) initMeshNetworks(args *PilotArgs, fileWatcher filewatcher.FileWatcher) {
+	if mw, ok := s.environment.Watcher.(mesh.NetworksWatcher); ok {
```

Comment:
How could meshConfig watcher be converted to NetworksWatcher

Reply:
We have one ConfigmapWatcher which reads the configmap. that configmap had both the mesh config and mesh networks